bin.src/movingObjects.py

```diff
@@ -0,0 +1,1535 @@
+#!/usr/bin/env python
+
+from __future__ import print_function, division
+
+import os
+import argparse
+import numpy as np
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+import lsst.sims.maf.metrics as metrics
+import lsst.sims.maf.slicers as slicers
+import lsst.sims.maf.stackers as stackers
+import lsst.sims.maf.plots as plots
+import lsst.sims.maf.db as db
+import lsst.sims.maf.metricBundles as mmb
+import lsst.sims.maf.utils as utils
+
+
+# Assumes you have already created observation file,
```

Comment:
Sounds like a good idea. I don't think I will do this for all of the metrics (we don't have notebooks for all of our other metrics, after all) but for some of them and to show what the plots looks like would be a good idea.